### DISCUSSION TITLE
Permission denie while upload photos to  Netbox  after upgradering till 3.6.1 from 3.5.7

### DISCUSSION THREAD

**Discussion by IoTTranas** [netbox-community/netbox#13748] [Help Wanted!]
Hi folk , 
 
Any Clue

Server Error
A file permission error was detected while processing this request. Common causes include the following:

Insufficient write permission to the media root. The configured media root is . Ensure that the user NetBox runs as has access to write files to all locations within this path.

The complete exception is provided below:

<class 'PermissionError'>

[Errno 13] Permission denied: '/opt/netbox/netbox/media/image-attachments/device_66_Vlan555.jpg'

Python version: 3.11.4
NetBox version: 3.6.1
Plugins: None installed
If further assistance is required, please post to the [NetBox discussion forum](https://github.com/netbox-community/netbox/discussions) on GitHub.

**Comment by candlerb** [comment]
"Insufficient write permission to the media root" => permissions not set correctly on /opt/netbox/netbox/media or its child directories.

See https://docs.netbox.dev/en/stable/installation/3-netbox/#create-the-netbox-system-user

**Comment by IoTTranas** [comment]
Hello there and thanks for quick responce,

I am running the Netbox with in Docker . How to solve the problem eith Netbox Dockers.??

Many thanks


//Mohammed Abood

Från: Brian Candler ***@***.***>
Skickat: den 13 september 2023 10:17
Till: netbox-community/netbox ***@***.***>
Kopia: Mohammed Abood ***@***.***>; Author ***@***.***>
Ämne: Re: [netbox-community/netbox] Permission denie while upload photos to Netbox after upgradering till 3.6.1 from 3.5.7 (Discussion #13748)

Var försiktig!
Detta mail kommer från en extern avsändare.
Var extra observant innan du öppnar länkar och bifogade filer, dessa kan innehålla virus. (Lämna aldrig ut några inloggningsuppgifter).

"Insufficient write permission to the media root" => permissions not set correctly on /opt/netbox/netbox/media or its child directories.

See https://docs.netbox.dev/en/stable/installation/3-netbox/#create-the-netbox-system-user

**Reply by candlerb** [reply]
You've given no information about your docker deployment. Are you perhaps running the docker-compose file from [here](https://github.com/netbox-community/netbox-docker/blob/release/docker-compose.yml) with docker-compose and docker CE? Or have you deployed it some other way?

If you're using their docker-compose then you could raise the issue in the [netbox-docker repository](https://github.com/netbox-community/netbox-docker/discussions). If you've deployed the netbox-docker container yourself then I think you'll need to sort out the permissions issues yourself.
